packages/tokens/src/_build/methods.ts

```
@@ -211,11 +233,11 @@ export async function createOutputFiles() {
           preprocessors: [
             'swisspost/box-shadow-keep-refs-workaround',
             'tokens-studio',
-            ...(config.proprocessors ?? []),
+            ...(config.preprocessors ?? []),
           ],
         });
 
-        config.platforms = Object.entries(config.platforms).reduce(
+        config.platforms = Object.entries(config.platforms || {}).reduce(
```

This would hide a bug if config.platforms were to not be defined. It might be better to throw an error if config.platforms isn't defined instead. What do you think?

Something like this beforehand:
`if (!config.platforms) return;`